packages/embed/src/components/dog-ear/DogEar.jsx

```diff
@@ -0,0 +1,25 @@
+import { IconCart } from '@audius/harmony'
+import cn from 'classnames'
+
+import Rectangle from '../../assets/img/dogEarRectangle.svg'
+
+import styles from './DogEar.module.css'
+
+export const DogEar = ({ size }) => {
+  return (
+    <>
+      <Rectangle
```

Comment:
I feel like this should be called something else? It's not _really_ a rectangle?

Reply:
+1, i copied this name from client... it's an isosceles triangle!

Reply:
renamed DogEar lol